src/main/test/base/NavigationTest.java

```diff
@@ -0,0 +1,87 @@
+package base;
+
+import org.junit.jupiter.api.Test;
+
+import base.constants.MapConstants;
+
+import base.navigationservice.NavigationService;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class NavigationTest {
+
+    @Test
+    public void TestCenter() {
+        //tests the getNextPortalToGetToCenter method
+        //method input - BottomLeftMap 
+        //expected output - BottomCenterMap
```

Comment:
Comments are redundant when you have meaningful variable names. Good practice is to name test method to match the method you are testing. For example, here you are testing `getNextPortalToGetToCenter` - so you can name test method `getNextPortalToGetToCenter_fromBottomLeftMap`, then you will not need first two lines of comments. Third line ( `expected output - BottomCenterMap` ) is also covered later with `String expected = MapConstants.BOTTOM_CENTER_MAP`